app/api/answers/get_answers.go

```diff
@@ -68,23 +104,43 @@ type rawAnswersData struct {
 }
 
 type answersResponseAnswerUser struct {
-	Login     string  `json:"login"`
+	// required: true
+	Login string `json:"login"`
+	// Nullable
 	FirstName *string `json:"first_name,omitempty"`
-	LastName  *string `json:"last_name,omitempty"`
+	// Nullable
+	LastName *string `json:"last_name,omitempty"`
 }
 
 type answersResponseAnswer struct {
-	ID             int64    `json:"id,string"`
-	Name           *string  `json:"name,omitempty"`
-	Type           string   `json:"type"`
-	LangProg       *string  `json:"lang_prog,omitempty"`
-	SubmissionDate string   `json:"submission_date"`
-	Score          *float32 `json:"score,omitempty"`
-	Validated      *bool    `json:"validated,omitempty"`
-
+	// required: true
+	ID int64 `json:"id,string"`
+	// Nullable
+	Name *string `json:"name,omitempty"`
+	// required: true
+	// enum: Submission,Saved,Current
+	Type string `json:"type"`
+	// Nullable
+	LangProg *string `json:"lang_prog,omitempty"`
+	// required: true
+	SubmissionDate string `json:"submission_date"`
+	// Nullable
+	Score *float32 `json:"score,omitempty"`
+	// Nullable
+	Validated *bool `json:"validated,omitempty"`
+
+	// required: true
 	User answersResponseAnswerUser `json:"user"`
 }
 
+// OK. Normal response of the itemAnswersView service
+// swagger:response itemAnswersViewResponse
+type itemAnswersViewResponse struct { // nolint:unused,deadcode
```

Comment:
You can put directly this construct in the service spec to avoid polluting the code with this. Cfr what I have written in the comments on top of `doc/errors.go`.

Reply:
discussed

Reply:
I didn't see it was a list, sorry.